components/06-examples/machtigen.hbs

```diff
@@ -0,0 +1,78 @@
+{{> '@header' }}
+<main>
+  <div>
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <div class="row">
+          <div class="col-md-12">
+            <h1>Inloggen</h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-12">
+            <ul class="nav nav-tabs">
+              <li class="active">
+                <a href="machtigen.html">Voor mezelf</a>
+              </li>
+              <li >
+                <a href="machtigen1.html">Voor een ander</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="row dso-rich-content">
+          <div class="col-sm-12">
+            <div class="dso-highlight-box dso-border" style="margin: 16px 0px 8px 0px"> 
+              <div class="row">
+                <div class="col-md-2 col-sm-2  col-xs-4 text-center"><img src="{{path '/images/Digid.png' }}" alt="Digid Logo" style="width:60px; height:60px;"></div>
```

Comment:
dubbel spatie tussen classnames

Reply:
En de `img` mag op een eigen regel: leest fijner en `div` is een block element waardoor je toch geen last hebt van whitespacing

Reply:
Daarbij is het good practice de kolom definities te sorteren op viewport: `col-xs-4 col-sm-2 col-md-2`.

Column definities cascaden naar grotere viewports. Je zet `col-sm-2`, je mag `col-md-2` dus achterwege laten: `col-xs-4 col-sm-2`

Overigens, zou je dit niet op een andere manier willen afvangen? Bootstrap kent het [Media component](https://getbootstrap.com/docs/3.3/components/#media). Scheelt weer kolom definities en de tekst lijnt netjes tegen het object aan

Reply:
En je kolommen komen niet uit op 12. En ze zijn te smal voor de inhoud:

![image](https://user-images.githubusercontent.com/9696261/72808366-b256a480-3c59-11ea-8a95-c0962ef8e5dd.png)